Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -3311,12 +3311,15 @@ public virtual bool ShowAllProperties
     }
 
     /// <summary>
-    ///   Gets or sets a flag that indicates whether to display an icon in front of the first value 
-    ///   column.
+    ///   Gets or sets a flag that indicates whether to display an icon in a <see cref="IBocColumnDefinitionWithRowIconSupport"/>.
     /// </summary>
+    /// <remarks>
+    ///   Which column the icon is displayed in is decided in the <see cref="BocColumnRendererArrayBuilder.CreateColumnRenderers"/>
```

Comment:
The implementation detail of `CreateColumnRenderers` is probably not helpful for a consumer. I would cut this information an just mention that the location is controlled by the `RowIconEnabled` flag on a column definition.

Usually when we reference some property of a type, for example, `a.b` we construct the XML docs in a similar fashion using `<see cref="a"/>.<see cref="a.b"/>` to have same "dot syntax" in the docs.